client/src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useContext } from "react";
+import { TouchableOpacity, Text, View, StyleSheet } from 'react-native';
+import api from "../services/api";
+import { SessionContext, IContextType } from "../services/session-context.service";
+import AuthService from "../services/auth.service";
+
+export const HomeScreen = () => {
+  const context = useContext<IContextType | null>(SessionContext);
+
+  const onSignOutPress = async () => {
+    await AuthService.signOut();
+    if (context !== null) context.deleteSession();
+  };
+
+  const onRefreshPress = () => {
+    api.get("/").then((res) => console.log('Refreshing..')).catch((err) => {
+      if (context !== null) context.deleteSession();
+    });
+  };
+
+  return (
+    <>
+      <View>
+        <TouchableOpacity style={styles.btn} onPress={onRefreshPress}>
+          <Text>רענן</Text>
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.btn} onPress={onSignOutPress}>
```

Comment:
We need to add something like a "Logout" icon to the action bar with the relevant functionality.
I looked at the mockup and it's quite stupid we decided to put the Logout button only in Settings screen..